src/main/groovy/org/gradle/android/workarounds/room/androidvariants/ApplyAndroidVariants.groovy

```diff
@@ -34,6 +34,16 @@ class ApplyAndroidVariants {
     }
 
     private static void applyToAllAndroidVariantsWithNewVariantApi(Project project, ConfigureVariants configureVariants) {
+        project.plugins.withId("com.android.application") {
+            configureNewVariants(project, configureVariants)
+        }
+
+        project.plugins.withId("com.android.library") {
```

Comment:
It's already merged, but shouldn't the plugin also apply to modules that use com.android.test plugin?

Reply:
I think it should also apply to modules with `com.android.dynamic-feature` plugin.

Reply:
@daio @eboudrant you're right. 
Checking the hierarchy of the plugins:
<img width="324" alt="Screen Shot 2023-01-15 at 10 38 40 AM" src="https://user-images.githubusercontent.com/475733/212560389-5f8d91c5-9a8b-4457-83df-35c792d3fffc.png">
I'm wondering if checking by `com.android.base` would be enough. Next week will work on this